test/unit/org/apache/cassandra/hints/HintsStoreTest.java

```diff
@@ -137,6 +138,26 @@ public void testConcurrentDeleteExpiredHints() throws Exception {
         assertFalse("All hints files should be deleted", store.hasFiles());
     }
 
+    @Test
+    public void testPendingHintsInfo() throws Exception
+    {
+        HintsStore store = HintsCatalog.load(directory, ImmutableMap.of()).get(hostId);
+        assertEquals(new PendingHintsInfo(store.hostId, 0, Long.MAX_VALUE, Long.MIN_VALUE),
+                     store.getPendingHintsInfo());
+
+        final long t1 = System.currentTimeMillis();
+        writeHints(directory, new HintsDescriptor(hostId, t1), 100, t1);
+        store = HintsCatalog.load(directory, ImmutableMap.of()).get(hostId);
```

Comment:
nit: Collections.emptyMap() and below too

Reply:
`HintsCatalog#load` expects `ImmutableMap`:

https://github.com/apache/cassandra/blob/b6877a5636a702fc045dfd8e1e4bbcf77ee8b959/src/java/org/apache/cassandra/hints/HintsCatalog.java#L65-L68

Reply:
Oh ok, makes sense then. Sorry for confusion!